int64 node_id
bool is_anchor
int64[] destination_id
float64[] distance
